src/modules/dashboard/queries.ts

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag';
```

Comment:
Not sure where this came from... maybe from a rebase?

Reply:
Yeah this file can go, just delete it